src-ui/stories/Page.svelte

```diff
@@ -1,6 +1,6 @@
 <script lang="ts">
   import './page.css'
-  import Header from './Header.svelte'
```

Comment:
this file is an auto-generated example, but the files it depends on (also an auto-generated example) was moved. Just minor upkeep while we haven't removed this file yet.